### ISSUE TITLE
Fix the Test and Validation set sizes

### ISSUE THREAD

**Issue by RafayAK** [alexhernandezgarcia/data-augmentation#6]
Currently, Test and Validation set sizes are also randomly generated. For comparing results across runs both the test and validation set sizes should be static. 

- Test set size should be 1000
- Validation set size should also be 1000

Sklearn datasets that should have static test and validation set sizes are:
- moons dataset
- circles dataset
- make blobs

**Comment by RafayAK** [comment]
https://github.com/alexhernandezgarcia/data-augmentation/pull/14
closes this issue
